common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -376,7 +376,9 @@
             onContainerMouseEnter: function (event) {
                 event.preventDefault();
                 $(event.currentTarget).find('.lang').addClass('is-opened');
-                this.state.el.trigger('language_menu:show');
+                if ($(event.currentTarget).find('.lang').length) {
```

Comment:
*deep inhale...*

Before the video player skin and functionality was updated several months ago, the language menu and transcript/language selection menu were one and the same. I added the 'CC' button and created a distinct button to open the menu. However since the menu applies to both the 'CC' and the Transcript, I've got the menu opening when a learner hovers over the 'CC' or Transcript buttons.

As a result, the event was firing even if there wasn't a menu (or more than one language) available. This conditional ensures the "open/close menu" event is only fired if there's more than one language and a menu is present.

This has been tested on a video with a single language and one with multiple languages and has been found to give us what we want.